deployment/params.py

```diff
@@ -633,6 +633,13 @@ def _deploy_proxy(
         return contract_type_container.at(proxy_contract.address)
 
     def upgrade(self, container: ContractContainer, proxy_address, data=b"") -> ContractInstance:
+        implementation = self.deploy(container)
+        # upgrade proxy to implementation
+        return self.upgradeProxyTo(implementation, proxy_address, data)
```

Comment:
Is this supposed to be:

```suggestion
        return self.upgradeTo(implementation, proxy_address, data)
```

Reply:
Not that I'm against the new function name if you want to change the name of the actual function.

Reply:
✔️